municipal_finance/static/javascript/table.js

```diff
@@ -741,13 +758,12 @@
       }
 
       // column (aggregate) headings
-      if (cube.columns.length > 1) {
+      if (cube.model.can_aggregate || cube.columns.length > 1) {
```

Comment:
I think let's use `if cube.name == 'capital_v2'` here instead.

The need for an additional row of column headings is a bit more complex than simply that _some_ column in the cube can not be aggregated. So let's not do logic based on can_aggregate yet.